sky/adaptors/cloudflare.py

```diff
@@ -119,3 +120,26 @@ def create_endpoint():
     endpoint = 'https://' + accountid + '.r2.cloudflarestorage.com'
 
     return endpoint
+
+
+def r2_is_enabled() -> bool:
+    """Checks if Cloudflare R2 is enabled"""
+
+    accountid_path = os.path.expanduser(ACCOUNT_ID_PATH)
+    return os.path.exists(accountid_path)
+
+
+def get_credential_file_mounts(file_mounts: Dict[str, str]) -> Dict[str, str]:
+    """Checks if aws credential file is set and update if not
+       Updates file containing account ID information
+
+    Args:
+        file_mounts: stores path to credential files of clouds
+    """
+
+    r2_credential_mounts = {}
+    if '~/.aws/credentials' not in file_mounts:
+        r2_credential_mounts.update(
+            {'~/.aws/credentials': '~/.aws/credentials'})
```

Comment:
I'm not fully sure why this check and update is necessary. Can't we simply return 
```
r2_credential_mounts = {'~/.aws/credentials': '~/.aws/credentials', ACCOUNT_ID_PATH: ACCOUNT_ID_PATH}
return r2_credential_mounts
```

Since the caller anyway calls dict.update(), adding `'~/.aws/credentials'` to the dictionary returned here should be idempotent.